shipstation_integration/items.py

```diff
@@ -73,6 +74,13 @@ def create_item(
 			}
 		)
 
+	if item.disabled:
+		# override disabled status to be able to add the item to the order
+		item.disabled = False
+		item.add_comment(
```

Comment:
@Alchez  Comment? No no, you wanna make it look like these kinds of updates- 

![image](https://user-images.githubusercontent.com/6697133/153177787-381995a9-e512-477f-91d4-bae0e405d0c3.png)

Reply:
Oh, I've changed the type now

![image](https://user-images.githubusercontent.com/13396535/153181072-a2789d84-2531-4fb1-89c4-dc09e63a22fb.png)